### PR TITLE
Add module doc for ssh_enumusers

### PR DESCRIPTION
**Pull request by wvu**
#10479

### CODE REVIEW

**Comment by wvu-r7** [comment]
# Release Notes

This adds documentation for the `auxiliary/scanner/ssh/ssh_enumusers` module.

File: documentation/modules/auxiliary/scanner/ssh/ssh_enumusers.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+Check for false positives (random username).
+
+## Usage
+
+```
+msf5 > use auxiliary/scanner/ssh/ssh_enumusers
+msf5 auxiliary(scanner/ssh/ssh_enumusers) > set rhosts [redacted]
+rhosts => [redacted]
+msf5 auxiliary(scanner/ssh/ssh_enumusers) > echo $'wvu\nbcook' > users
```
Comment (h00die) [inline documentation/modules/auxiliary/scanner/ssh/ssh_enumusers.md]: i like this trickyness for doc purproses

Comment (wvu-r7) [reply documentation/modules/auxiliary/scanner/ssh/ssh_enumusers.md]: It's laziness that still shows what I'm doing. :P

Comment (busterb) [reply documentation/modules/auxiliary/scanner/ssh/ssh_enumusers.md]: Mr. One-liner

**Pull request merged**